pkg/apis/compliance/v1alpha1/compliancescan_types.go

```diff
@@ -367,6 +372,19 @@ func (cs *ComplianceScan) GetScanTypeIfValid() (ComplianceScanType, error) {
 	return "", ErrUnkownScanType
 }
 
+// GetScanerTypeIfValid returns scanner type we will be using if the scan has a valid one, else it returns
+// an error
+func (cs *ComplianceScan) GetScanerTypeIfValid() (ScannerType, error) {
```

Comment:
nit: Could steal a leaf from the kubeclient book and name if `GetScannerTypeOrDie`) - or we could use `OrFail` if we remove the `panic(err)` eventually.